src/Stratis.Bitcoin.Features.ColdStaking.Tests/ColdStakingControllerTest.cs

```diff
@@ -155,8 +157,34 @@ private void CreateMempoolManager()
 
             // Create mempool validator.
             var mempoolLock = new MempoolSchedulerLock();
+
+            // The mempool rule constructors aren't parameterless, so we have to manually inject the dependencies for every rule
+            var mempoolRules = new List<MempoolRule>
```

Comment:
why do you need o do this the rules should be present on the network

Reply:
ah this is to create eh instances of the rules, wont be easy to mock this.

Reply:
Yeah, it's ugly, but as the rule constructors need parameters I'm not sure how to do it in a better way without introducing some sort of auto-DI for the tests, which seems unnecessarily complex.